wandb/sdk/wandb_settings.py

```diff
@@ -116,7 +116,10 @@ def _get_program() -> Optional[Any]:
     try:
         import __main__  # type: ignore
 
-        return __main__.__file__
+        if __main__.__spec__ is None:
+            return __main__.__file__
+        # likely run as `python -m ...`
+        return f"{sys.executable} -m {__main__.__spec__.name}"
```

Comment:
Shouldn't this be simply `f"{__main__.__spec__.name}"`?

Reply:
yeah, i think "-m ... " makes sense to me
that seems most consistent with other cases.